templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a class="btn btn-primary" style="float: right" href="{% url 'taxi:manufacturer-create' %}" role="button">Create</a>
```

Comment:
Ensure there is consistent spacing around the attributes in the `<a>` tag. There is an extra space before the `class` attribute. Consider removing this extra space for consistency and to adhere to HTML best practices.